stable/ambassador/templates/deployment.yaml

```diff
@@ -183,3 +183,6 @@ spec:
     {{- end }}
       imagePullSecrets:
         {{- toYaml .Values.imagePullSecrets | nindent 8 }}
+      dnsPolicy: {{ .Values.dnsPolicy | default "ClusterFirst" }}
+      hostNetwork: {{ .Values.hostNetwork | default "false" }}
```

Comment:
set the defaults in `values.yaml` instead to be consistent with other values.

Reply:
Will do, sorry I missed that.